influx_logging/handler.py

```diff
@@ -79,26 +79,26 @@ def __init__(self,
         self.indexed_keys = {'level','short_message'}
         self.client = InfluxDBClient(database=database, **client_kwargs)
         self.backpop = backpop
-        
+
         if database not in {x['name'] for x in self.client.get_list_database()}:
             self.client.create_database(database)
-        
+
         if indexed_keys is not None:
-            self.indexed_keys += set(indexed_keys)
-        
+            self.indexed_keys |= set(indexed_keys)
```

Comment:
I also fixed a bug where sets were being added together with `+` when they should have been using `|`. The `+` operand is not a supported operand between sets and throws an error in both Python 2 and Python 3. So I'm not sure if the `indexed_keys` configuration bit ever actually worked...